pkg/sypgp/sypgp.go

```diff
@@ -688,7 +739,10 @@ func ExportPrivateKey(kpath string, armor bool) error {
 		err = entityToExport.SerializePrivate(file, nil)
 	} else {
 		var keyText string
-		keyText, err = serializeEntity(entityToExport, openpgp.PrivateKeyType)
+		keyText, err = serializePrivateEntity(entityToExport, openpgp.PrivateKeyType)
+		if err != nil {
+			sylog.Fatalf("Failed to read private key ASCII armored format: %s\n", err)
```

Comment:
Again, we should return a error, instead of exiting right here.

```suggestion
			return fmt.Errorf("failed to read ASCII key format: %s", err)
```